aws/terminator/networking.py

```diff
@@ -499,3 +499,66 @@ def name(self):
 
     def terminate(self):
         self.client.delete_rule_group(RuleGroupArn=self.id)
+
+
+class TransitGateway(Terminator):
```

Comment:
There's currently something in compute.py for both of these (I put the attachment piece there because the TGW piece was there).

Probably worth deleting the ones from compute.py

Reply:
ah I totally missed that - thanks!  I'll remove whatever is redudant